README.md

```diff
@@ -45,113 +41,136 @@ $ ... build your other MySQL flavors/forks here in the same way, modifying the b
 ```
 # What options does pstress accept?
 
-First, take a quick look at ``` pstress --help, pstress --config-help, pstress --cli-help, pstress --help --verbose ``` to see available modes and options.
+First, take a quick look at ``` ./pstress-ps --help, ./pstress-ps --help --verbose ``` to see available modes and options.
 
 # Command line options example:
 
 Option | Function | Example | Default
 --- | --- | --- | ---
---seed | Initial seed used for the test | --seed=1001 | Random value
---tbs-count | random number of different general tablespaces | --tbs-count=3 | default#: 1
---undo-tbs-count | Number of default undo tablespaces | --undo-tbs-count=3 | default#: 3
---undo-tbs-sql | Assign probability of running create/alter/drop undo tablespace | --undo-tbs-sql=50 | default#: 1
---engine | Engine used | --engine=InnoDB | default: INNODB
---jlddl | load DDL and exit | --jlddl | default: 0
---no-ddl | do not use ddl in workload | --no-ddl | default: 0
---only-cl-ddl | only run command line ddl. other ddl will be disabled | --only-cl-ddl | default: 0
---only-cl-sql | only run command line sql. other sql will be disable | --only-cl-sql | default: 0
---no-encryption | Disable All type of encryption | --no-encryption | default: 0
---encryption-type | all ==> keyring/Y/N | --encryption-type=keyring | default: Y/N
---no-column-compression | Disable column compression. It is percona style compression | | default: 0
---no-table-compression | Disable table compression | | default: 0
---no-tbs | disable all type of tablespace including the general tablespace | | default: 0
---no-blob | Disable blob columns | | default: 0
---no-virtual | Disable virtual columns | | default: 0
---tables | Number of initial tables | --tables=10 | default#: 10
---indexes | maximum indexes in a table,default depends on page-size as well | --indexes=2 | default#: 7
+--add-index | alter table add random index | | default#: 1
+--add-drop-partition | randomly add drop new partition | | default#: 3
+--add-column | alter table add some random column | --add-column=10 | default#: 1
+--address | IP address to connect to | | default: 
 --alter-algorith | algorithm used in alter table. INPLACE/COPY/DEFAULT/ALL | --alter-algorith=INPLACE | default: all

```

Comment:
Please use this command to see the correct sorting order
./pstress-ps --help --verbose | grep "^\-\-" | sort